psi4/driver/driver_cbs.py

```diff
@@ -871,392 +996,59 @@ def cbs(func, label, **kwargs):
-    tables += table_delimit
-    tables += """     %6s %20s %1s %-27s %2s %16.8f   %-s\n""" % (
-        GRAND_NEED[0]['d_stage'], GRAND_NEED[0]['d_wfn'], '/', GRAND_NEED[0]['d_basis'], '', GRAND_NEED[0]['d_energy'],
-        GRAND_NEED[0]['d_scheme'])
-    if len(metadata) > 1:
-        tables += """     %6s %20s %1s %-27s %2s %16.8f   %-s\n""" % (
-            GRAND_NEED[1]['d_stage'], GRAND_NEED[1]['d_wfn'], '/', GRAND_NEED[1]['d_basis'], '',
-            GRAND_NEED[1]['d_energy'] - GRAND_NEED[2]['d_energy'], GRAND_NEED[1]['d_scheme'])
-    if len(metadata) > 2:
-        dc = 3
-        for delta in metadata[2:]:
-            deltaE_total = GRAND_NEED[dc]['d_energy'] - GRAND_NEED[dc + 1]['d_energy']
-            tables += """     %6s %20s %1s %-27s %2s %16.8f   %-s\n""" % (
-                GRAND_NEED[dc]['d_stage'], GRAND_NEED[dc]['d_wfn'] + ' - ' + GRAND_NEED[dc + 1]['d_wfn'], '/',
-                GRAND_NEED[dc]['d_basis'], '', deltaE_total,
-                GRAND_NEED[dc]['d_scheme'])
-            core.set_variable(f"CBS {GRAND_NEED[dc]['d_stage'].upper()} TOTAL ENERGY", deltaE_total)
-            dc += 2
-
-    tables += """     %6s %20s %1s %-27s %2s %16.8f   %-s\n""" % ('total', 'CBS', '', '', '', finalenergy, '')
-    tables += table_delimit
-
-    core.print_out(tables)
-
-    core.set_variable('CBS REFERENCE ENERGY', GRAND_NEED[0]['d_energy'])
-    core.set_variable('CBS CORRELATION ENERGY', finalenergy - GRAND_NEED[0]['d_energy'])
-    core.set_variable('CBS TOTAL ENERGY', finalenergy)
-    core.set_variable('CURRENT REFERENCE ENERGY', GRAND_NEED[0]['d_energy'])
-    core.set_variable('CURRENT CORRELATION ENERGY', finalenergy - GRAND_NEED[0]['d_energy'])
-    core.set_variable('CURRENT ENERGY', finalenergy)
-    core.set_variable('CBS NUMBER', Njobs)
-
-    # new skeleton wavefunction w/mol, highest-SCF basis (just to choose one), & not energy
-    basis = core.BasisSet.build(molecule, "ORBITAL", 'def2-svp')
-    wfn = core.Wavefunction(molecule, basis)
+    pass
 
-    optstash.restore()
 
-    if ptype == 'energy':
-        finalquantity = finalenergy
-    elif ptype == 'gradient':
-        finalquantity = finalgradient
-        wfn.set_gradient(finalquantity)
-        if finalquantity.rows(0) < 20:
-            core.print_out('CURRENT GRADIENT')
-            finalquantity.print_out()
-    elif ptype == 'hessian':
-        finalquantity = finalhessian
-        wfn.set_gradient(finalgradient)
-        wfn.set_hessian(finalquantity)
-        if finalquantity.rows(0) < 20:
-            core.print_out('CURRENT HESSIAN')
-            finalquantity.print_out()
-
-    if return_wfn:
-        return (finalquantity, wfn)
-    else:
-        return finalquantity
+##  Aliases  ##
+complete_basis_set = cbs
 
+######### OUTSOURCE
+######### PARSE / PREPARE
+######### PREPARE / COMPUTE
 
-######### COMPUTE / ASSEMBLE
-######### ASSEMBLE / REPORT
+#    #psioh = core.IOManager.shared_object()
```

Comment:
What is this commented out section doing here?

Reply:
details that didn't make the transition. now notated as such.